.github/workflows/python-package.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
         pip-version: ['', '==21.2.4']
```

Comment:
`pip-version` matrix setting was introduced to guarantee the operation of multiple pip versions.

Your contribution will make this setup unnecessary.

We would like to see the YAML configuration file changed as follows

As-Is:
```yaml
    strategy:
      matrix:
        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
        pip-version: ['', '==21.2.4']
# snip
    - name: Install dependencies
      run: |
        python -m pip install --upgrade pip${{ matrix.pip-version }}
```

To-Be:
```yaml
    strategy:
      matrix:
        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
# snip
    - name: Install dependencies
      run: |
        python -m pip install --upgrade pip
```